Infrastructure/mosquitto/build-dtc-mosqtuitto.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
For Linux & macOS users, it would be very convenient if this file had the `executable` bit set.

Reply:
Hmmm. When I look at the files in the Git Bash shell, the exec bit seems to be set on the script:

![image](https://user-images.githubusercontent.com/1908205/133324330-f14a3cf7-5d0f-4ddf-840e-80ba7dff45f1.png)

Let's discuss this offline to see what's going on here.